dicom-archive/updateMRI_Upload.pl

```diff
@@ -3,6 +3,87 @@
 # zia.mohades@mcgill.ca
 # Perl tool to update the mri_upload table
 
+=pod
+
+=head1 NAME
+
+updateMRI_Upload.pl - updates database table mri_upload according to an entry in table
+   tarchive
+
+=head1 SYNOPSIS
+
+updateMRI_Upload.pl [options] -profile prod -tarchivePath tarchivePath -source_location source_location
+
+=over 2
+
+=item *
+B<-profile prod> : (mandatory) path (absolute or relative to the current directory) of the 
+    profile file
+
+=item *
+B<-tarchivePath tarchivePath> : (mandatory) absolute path to the tarchive file
+
+=item *
+B<-source_location source_location> : (mandatory) value to set column 
+    DecompressedLocation to for the newly created record in table mri_upload (see below)
```

Comment:
Put DecompressedLocation and mri_upload in code format

Reply:
done